sdk/python/kfp/dsl/_container_op.py

```diff
@@ -1011,19 +1011,19 @@ def foo_pipeline(tag: str, pull_image_policy: str):
     _DISABLE_REUSABLE_COMPONENT_WARNING = False
 
     def __init__(
-      self,
-      name: str,
-      image: str,
-      command: StringOrStringList = None,
-      arguments: StringOrStringList = None,
-      init_containers: List[UserContainer] = None,
-      sidecars: List[Sidecar] = None,
-      container_kwargs: Dict = None,
-      artifact_argument_paths: List[InputArgumentPath] = None,
-      file_outputs: Dict[str, str] = None,
-      output_artifact_paths: Dict[str, str]=None,
-      is_exit_handler=False,
-      pvolumes: Dict[str, V1Volume] = None,
+        self,
+        name: str,
+        image: str,
+        command: StringOrStringList = None,
+        arguments: StringOrStringList = None,
+        init_containers: List[UserContainer] = None,
+        sidecars: List[Sidecar] = None,
+        container_kwargs: Dict = None,
+        artifact_argument_paths: List[InputArgumentPath] = None,
+        file_outputs: Dict[str, str] = None,
+        output_artifact_paths: Dict[str, str] = None,
+        is_exit_handler=False,
+        pvolumes: Dict[str, V1Volume] = None,
     ):
```

Comment:
Optionally revert this file since there's only formatting change? Otherwise, since you're touching this, can you help fix all the argument type hints to be `Optional[...] = None`?

Reply:
Good catch. Added `Optional`